### PR TITLE
Refresh TDM mapcycle

### PR DESCRIPTION
**Pull request by NoahTheLegend**
## Status

- **READY**: this PR is (to the best of your knowledge) ready to be incorporated into the game.

## Description

	HeraldsMarshes.png; 				# removed, outdated, archers do stalemate here
	CrampedCastle.png; 					# removed, outdated
	Amarok_Revised.png;					# removed, outdated, too rough topology, can be flooded
	Ommadawn_Revised.png;				# removed, outdated, bad mid design
	Cloister.png; 						# removed, outdated, has a stalemate spot on upper mid (if you bombjump into open cage under ceiling)
	Labyrinth.png;						# removed, outdated
	BorgRuins.png;						# removed, shitty traps + death pit with chests sink mid in chaos (and teamkills)
	FortMercury_Revised.png;			# removed, outdated
	LavalordTDM1.png; 					# removed, outdated, some spots are unbalanced (i.e. mid holes, roof above entries in bottom mid)
	GurinTDM3.png; 						# removed, outdated
	Biurza_Black_Dog.png; 				# removed, slightly outdated, almost has no strategical passages, archers do stalemate here
	Biurza_Abrupt_Ending.png;   		# removed, mid design is weird, pushing side has no advantages anywhere
	Biurza_Death_Penalty.png;   		# removed, pretty huge, outdated, gameplay narrows to same spot and goes same time every match
	Ni_Ashes.png; 						# removed, outdated, too huge, rough topology disallows getting to mid quickly
	Ferrezinhre_Wish.png; 				# removed, slightly outdated, very serious map with rough and open topology, not sure about decision
	8x_Dirtkissing.png; 				# removed, outdated, has a spot for stalemate\camp on middle platform
	8x_Cascade1.png; 					# removed, outdated and boring (most of passages don't allow sudden attacks also)
	8x_TDM1.png; 						# removed, hard topology for amateur knights, played mostly with using bombs \ jumping or climbing through holes
	super_gaming_geek_ChokePoint.png;   # removed, too huge, gameplay narrows to the bottom in 8 of 10 matches
	Yeti5000_Upsurge_Revised.png; 		# outdated, replaced with earlier version

 PressureCooker.png; 					# funny map, open bottom mid, has spots to defend from upside
 MonsteriTDM1.png; 					# added, good map for archers, nostalgy
 MonsteriTDM2.png; 					# added, simple and quick map, however archers can stalemate here
 Dichotomy.png; 						# added, nostalgy + people liked it
 Bastille.png; 						# added, quick map, good mid design
 ArchPoint.png; 						# added, funny map
 HergestRidge.png; 					# added, old map, nostalgy, removed trampolines, added another passage to middle on right side, shrinked right side so its more symmetric, made side walls smaller
 Biurza_Spotlight_Kid.png; 			# added, nostalgy, sometimes funny
 Biurza_Cock_Fight.png;				# added, nostalgy, sometimes funny, has water fights sometimes, edited: sunken trampolines into ground# Ni_GetisArena.png; 					# added, not sure about this, just nostalgy? iirc has a stalemate if you explode a bomb under gates
 Ferrezinhre_CursedWell.png; 			# added, nostalgy, sometimes funny enough, slightly edited passages from spawns to mid
 FG_Descent.png; 						# added, funny map, good design of middle (that collapses though)
 FG_Break.png; 						# added, nostalgy, however i dislike bottom side
 FG_Enclosure.png; 					# added, weird but pretty funny map
 FG_SpaceInvader.png; 					# added, edited chests positions, added floor passages from mid to bottom, removed trapblocks on mid side, made a hole in doors
 bunnie_Wojciech_Jaruzelski.png; 		# added, old map, slightly updated (added passage from spawn to lever)
 Vans_Spelunca.png; 					# added, no complex mid design (doesnt mean its bad), has mechanisms, tough and quick small map
 Vans_Corruptio.png; 					# added, very old map, edited moderately: shop&spawn pos replaced, narrowed waterfall, removed 3rd block in lower passages so slashing people jumping down is easier, made symmetric, blocked mid passage with platforms
 Vans_Ziraeth.png; 					# added, very old map, edited a lot: removed ponds, made map shorter for 1/3 (was 151 in width, now 103), made climbing to upperside easier, removed platforms above void and archer camp spots, gameplay on it should me more quick now
 Pineapple_Vestige.png; 				# added, good platformer map design
 GurinTDM1.png; 						# added, nostalgy
 GurinTDM2.png; 						# added, nostalgy, added shops
 Eluded_JoansMansion.png; 				# added, quick map, has a lot of spots to outplay people, however quite serious
 CrocoDile_FireBird.png;				# added, good platforming
 Cohen_Sanctuary.png; 					# added, nostalgy, quick map
 SJD360_CrowbarsSwitchblades.png;		# added, very old map, slightly revised design, slightly edited rough terrain, removed archer camp spots, added a shop in the middle
 mcrifel_Ruin.png; 					# added, very old map, neat and small, slightly edited corners, made more symmetric and fair for pushing a cornercamp

### CODE REVIEW

File: Maps/Official/TDM/Biurza_Cock_Fight.png
Comment (AsuMagic) [inline Maps/Official/TDM/Biurza_Cock_Fight.png:1]: Should be added back in repo

File: Maps/Official/TDM/Ferrezinhre_CursedWell.png
Comment (AsuMagic) [inline Maps/Official/TDM/Ferrezinhre_CursedWell.png:1]: Should be added back in repo

File: Rules/TDM/mapcycle.cfg [hunk 1]
```diff
@@ -1,61 +1,79 @@
...
+				# JoanOfArc
+			Maps/Official/TDM/Eluded_JoansMansion.png;
+				# CrocoDile
+			Maps/Official/TDM/CrocoDile_FireBird.png;
+				# Cohen
+			Maps/Official/TDM/Cohen_Sanctuary.png;
+				# SJD360
+			Maps/Official/TDM/SJD360_CrowbarsSwitchblades.png; 
+				# mcrifel
+			Maps/Official/TDM/mcrifel_Ruin.png;
```
Comment (AsuMagic) [inline Rules/TDM/mapcycle.cfg:79]: None of the `_v2` variants are being used. The `gamemode.cfg` should be edited.

**Review by AsuMagic — CHANGES_REQUESTED** [review]

File: Maps/Official/TDM/Ferrezinhre_CursedWell.png
Comment (NoahTheLegend) [reply Maps/Official/TDM/Ferrezinhre_CursedWell.png:1]: These maps are stored in Maps/RTDM, the _v2 edited versions are stored in Official mapcycle. I do not get what you want me to do

File: Rules/TDM/mapcycle.cfg (hunk 1 above)
Comment (NoahTheLegend) [reply Rules/TDM/mapcycle.cfg:79]: you meant `mapcycle.cfg`?

Comment (AsuMagic) [reply Rules/TDM/mapcycle.cfg:79]: yes, whoops

File: Maps/Official/TDM/Ferrezinhre_CursedWell.png
Comment (AsuMagic) [reply Maps/Official/TDM/Ferrezinhre_CursedWell.png:1]: ah, ok, I didn't realize those were duplicate in the game files in the first place.

File: Rules/TDM/mapcycle.cfg [hunk 2]
```diff
@@ -1,61 +1,79 @@
...
-			Maps/Official/TDM/Ferrezinhre_Wish.png;
+			Maps/Official/TDM/Ferrezinhre_CursedWell_v2.png;
 				# 8x
 			Maps/Official/TDM/8x_Virgo.png;
-			Maps/Official/TDM/8x_Dirtkissing.png;
-			Maps/Official/TDM/8x_Cascade1.png;
 			Maps/Official/TDM/8x_Mandala.png;
-			Maps/Official/TDM/8x_TDM1.png;
-			Maps/Official/TDM/8x_TDM2.png;
+			Maps/Official/TDM/8x_TDM2_v2.png;
```
Comment (AsuMagic) [inline Rules/TDM/mapcycle.cfg:43]:
```suggestion
			Maps/Official/TDM/8x_TDM2.png;
```

File: Rules/TDM/mapcycle.cfg [hunk 3]
```diff
@@ -1,61 +1,79 @@
...
+			Maps/Official/TDM/Yeti5000_Upsurge.png;
+				# Vans
+			Maps/Official/TDM/Vans_Spelunca.png;
+			Maps/Official/TDM/Vans_Corruptio_v2.png;
+			Maps/Official/TDM/Vans_Ziraeth_v2.png;
+				# Pineapple
+			Maps/Official/TDM/Pineapple_Vestige.png;
+				# Gurin
+			Maps/Official/TDM/GurinTDM1.png;
+			Maps/Official/TDM/GurinTDM2.png;
```
Comment (AsuMagic) [inline Rules/TDM/mapcycle.cfg:69]:
```suggestion
			Maps/Official/TDM/GurinTDM2_v2.png;
```

**Review by AsuMagic — APPROVED** [review]

**Comment by AsuMagic** [comment]
Maps LGTM

**Pull request merged**